# reason:
#    since bebf09a3b33b72027a0a0601f2b331d524630ad6 qt5-rcc defaults to
#    zstd compression which is unsupprted for format-version<3
#    see: https://travis-ci.org/Piraty/void-packages/jobs/562765059#L5398-L5411
# upstream: no

--- a/ui/qt/CMakeLists.txt
+++ b/ui/qt/CMakeLists.txt
@@ -601,7 +601,7 @@
 	# so this has no unwanted side effects. This mtime started appearing in
 	# Qt 5.8. The option to force the old file format without mtime was
 	# added in Qt 5.9. See https://bugreports.qt.io/browse/QTBUG-58769
-	set(CMAKE_AUTORCC_OPTIONS --format-version 1)
+	set(CMAKE_AUTORCC_OPTIONS --format-version 1 --compress-algo zlib)
 endif()
 
 QT5_ADD_TRANSLATION(WIRESHARK_QT_QM ${WIRESHARK_QT_TS})
